configuration/dto.go

```diff
@@ -41,25 +41,17 @@ type RealmConfiguration struct {
 
 // RealmAdminConfiguration struct
 type RealmAdminConfiguration struct {
-	Mode                                  *string                   `json:"mode"`
-	AvailableChecks                       map[string]bool           `json:"available-checks,omitempty"`
-	Accreditations                        []RealmAdminAccreditation `json:"accreditations,omitempty"`
-	SelfRegisterEnabled                   *bool                     `json:"self_register_enabled"`
-	Theme                                 *string                   `json:"theme,omitempty"`
-	NeedVerifiedContact                   *bool                     `json:"need_verified_contact,omitempty"`
-	ConsentRequiredSocial                 *bool                     `json:"consent_required,omitempty"`
-	ConsentRequiredCorporate              *bool                     `json:"consent_required_corp,omitempty"`
-	ShowGlnEditing                        *bool                     `json:"show_gln_editing,omitempty"`
-	VoucherEnabled                        *bool                     `json:"voucher_enabled"`
-	VideoIdentificationAccountingEnabled  *bool                     `json:"video_identification_accounting_enabled"`
-	VideoIdentificationPrepaymentRequired *bool                     `json:"video_identification_prepayment_required"`
-}
-
-// RealmAdminAccreditation struct
-type RealmAdminAccreditation struct {
-	Type      *string `json:"type,omitempty"`
-	Validity  *string `json:"validity,omitempty"`
-	Condition *string `json:"condition,omitempty"`
+	Mode                                  *string         `json:"mode"`
+	AvailableChecks                       map[string]bool `json:"available-checks,omitempty"`
```

Comment:
If accreditations are already stored in DB, won't this change make the json.unmarshall(...) fail?
I think we should force the field to nil after deserialization or configure the unmarshaller to ignore unknown fields...

Reply:
It will not crash unmarshaller ignore unknown fields by default.